cmd/argocd/commands/app.go

```diff
@@ -997,6 +1001,7 @@ func NewApplicationDeleteCommand(clientOpts *argocdclient.ClientOptions) *cobra.
 		},
 	}
 	command.Flags().BoolVar(&cascade, "cascade", true, "Perform a cascaded deletion of all application resources")
+	command.Flags().StringVarP(&propagationPolicy, "propagation-policy", "p", "foreground", "Specify propagation policy for deletion of application resources")
```

Comment:
It would be helpful to list possible values (foreground and background) in the command help.

Reply:
The default value will be displayed by the cobra. Will add `One of: foreground|background`

Flag message:
```shell
Specify propagation policy for deletion of application resources. One of: foreground|background (default "foreground")
```